.travis.yml

```diff
@@ -1,8 +1,13 @@
-sudo: false
 language: go
+go_import_path: github.com/twitchtv/retool
+
 go:
+- 1.7.5
 - 1.8
-go_import_path: github.com/twitchtv/retool
+- tip
+
 script:
+- go get github.com/alecthomas/gometalinter
```

Comment:
I thought of that, but not sure if that is circular? Also with gml vendoring it's own tools is that easy?

Alternatively we can use retool on the individual linters and not use gml?